modules/recieve_statustxt.py

```diff
@@ -0,0 +1,63 @@
+import time
+from pymavlink import mavutil
+from modules.common.data_encoding.message_encoding_decoding import decode_bytes_to_position_global
+from modules.common.data_encoding.metadata_encoding_decoding import decode_metadata
+from modules.common.data_encoding.worker_enum import WorkerEnum
+from modules.common.kml.kml_conversion import positions_to_kml
+from pathlib import Path
```

Comment:
Our import convention is to have Python imports first, then outside library imports, then our own module imports. There is a space between each section. We only import modules (files), so no importing classes or functions directly (ie no `import WorkerEnum`, instead import `worker_enum`). Finally, everything needs to be in alphabetical order, and two spaces after all of the imports

Reply:
Done :)